connector.yaml

```diff
@@ -0,0 +1,162 @@
+version: "1.0"
+specification:
+  name: connectorname
+  summary: <describe your connector>
+  description: <describe your connector in detail>
+  version: (devel)
```

Comment:
```suggestion
  version: <version>
```

Reply:
I think I'll need to replace < and > everywhere, since it looks like Docusaurus doesn't like it.